Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -76,6 +84,8 @@ final class PledgePaymentPlanOptionView: UIView {
       ),
       for: .normal
     )
+
+    self.ineligibleBadgeLabel.text = "Available for pledges over $150"
```

Comment:
Two thoughts:
 
 1. This $150 - or even better, the entire badge text! - should come from the backend. It seems likely the eligibility requirements are something that will change as the project evolves.
 2. Needs a `//TODO` comment for translations so we don't forget it